public/js/monitor.js

```diff
@@ -95,25 +124,39 @@ async function sha1(message) {
 
 async function hashEmailAndSend(emailFormSubmitEvent) {
   emailFormSubmitEvent.preventDefault();
-  handleEvents("Scan");
   const emailForm = emailFormSubmitEvent.target;
+  if(emailForm.querySelector("input[name=email]").value === ""){
```

Comment:
Not sure if you want to `.trim()` the string in case it has whitespace, but probably not important one way or another. Not sure how this'd work w/ a clearly bogus email, or email adjacent looking item, like "foo-bar" or "foo@bar".

Reply:
Clearly bogus emails (without a '.com' attached to them at least) and spaces get caught by enableBtnIfEmailValid() and never make it that far. This is just for catching an empty form submission.